README.md

```diff
@@ -591,6 +591,101 @@ Possible errors are:
 - 400 - Already rolled back - if redemption with given `redemption_id` has been rolled back already
 - 400 - Invalid redemption id - when trying to rollback a rollback.
 
+
+#### Create customer
+
+Example:
+
+```python
+payload = {
+    "name": "Melanie Morgan",
+    "email": "melanie@email.com",
+    "description": "Description of Melanie personality for test reason",
+    "metadata": {
+        "lang": "en"
+    }
+}
+
+result = voucherify.customer.create(payload)
+```
+
+Result:
+```json
+{
+    "id": "cust_WGG615E92dhOHz7PV9Vo9gk9",
+    "created_at": "2016-06-12T15:52:49Z",
+    "description": "Description of Melanie personality for test reason",
+    "email": "melanie@email.com",
+    "metadata": {
+        "lang": "en"
+    },
+    "name": "Melanie Morgan",
+    "object": "customer"
+}
+```
+
+#### Fetch customer
+
+Example:
+
+```python
+result = voucherify.customer.fetch("cust_gVYAaioitMz3GO6HSKFLf7Or")
+```
+
+Result:
+```json
+{
+    "id": "cust_gVYAaioitMz3GO6HSKFLf7Or",
+    "created_at": "2016-06-12T16:03:36Z",
+    "description": "Description of Melanie personality for test reason",
+    "email": "melanie@email.com",
+    "metadata": {
+        "lang": "en"
+    },
+    "name": "Melanie Morgan",
+    "object": "customer"
+ }
+```
+
+#### Update customer
+
+Example:
+
+```python
+payload = {
+    "id": "cust_gVYAaioitMz3GO6HSKFLf7Or",
+    "description": "Updated description for given customer ID"
+}
+
+result = voucherify.customer.update(payload)
+```
+
+Result:
+```json
+{
+    "id": "cust_gVYAaioitMz3GO6HSKFLf7Or",
+    "created_at": "2016-06-12T16:03:36Z",
+    "description": "Improved description about Melanie personality",
+    "email": "melanie@email.com",
+    "metadata": {
+        "lang": "en"
+    },
+    "name": "Melanie Morgan",
+    "object": "customer"
+ }
+```
+
+#### Delete customer
+
+Example:
+
+```python
+result = voucherify.customer.delete("cust_gVYAaioitMz3GO6HSKFLf7Or")
+```
+
+Result:
+`This method doesn't return result`
```

Comment:
technically, the result is an empty message body.

btw this is inconsistent with https://github.com/rspective/voucherify-nodejs-sdk#delete-customer